plugins/catalog-backend-module-gitlab/src/lib/types.ts

```diff
@@ -61,7 +61,16 @@ export type GitlabProviderConfig = {
   host: string;
   group: string;
   id: string;
-  branch: string;
+  /**
+   * The name of the branch to be used, to search for catalog-info
```

Comment:
```suggestion
   * The name of the branch to be used, to search for catalog files
```

It's not strictly required to name them catalog-info